content/docs/use-cases/fast-data-storage-layer.md

```diff
@@ -0,0 +1,71 @@
+# Fast Data Storage Layer
+
+Datasets used in data science tend to exceed typical storage and networking
+capacities. Storage needs expand rapidly as more people acquire the same data,
+creating duplication (increasing cost). Valuable time is wasted waiting for
+downloads in each environment.
+
+![](/img/dataset-copies.png) _Slow synchronization of data storage_
+
+DVC's built-in data <abbr>caching</abbr> lets you implement a simple and
+efficient storage layer globally -- for your entire team. This approach can help
+to
+
+- speed up data transfers from massive object stores currently on the cloud, or
+  move a growing dataset out of your machine without slowing things down.
+
+- pay only for fast access to frequently-used data (upgrading your entire
+  storage platform is expensive).
+
+- de-duplicate files automatically when multiple people are working on the same
+  data (for example on a
+  [shared development server](#example-shared-development-server)).
+
+- switch data inputs quickly (without re-downloading) on a shared GPU server for
+  machine learning [experiments](/doc/user-guide/experiment-management).
+
+![](/img/storage-layers.png) _Data storage middleware for multiple projects_
```

Comment:
I think I mentioned this last review, but the "Remote" column here includes some confusing items like SSH and EFS. IMO this column should repeat the "External" column above - SFTP, S3, GCP, Azure (and maybe the columns should be named the same in both diagrams).

Also, should the caption include "fast data storage layer" to make it clear that this is what the title refers to?

Reply:
> IMO this column should repeat the "External" column above

Agree. I was just lazy to update that image haha. Will take care in https://github.com/iterative/dvc.org/issues/2677 (noted your comments there).

> should the caption include "fast data storage layer" to make it clear

Hmmm maybe. I hope it can be pretty evident though. Maybe just "fast layer" in the bottom (green text currently)?

Reply:
UPDATE: I did update the 2nd figure now (still a draft though).

Reply:
Also, I'm not sure that "External" is the best term in the 2nd image... The "Near" layer can be external too. Maybe should be "Remote" in both figures?